app/src/main/java/org/wikipedia/views/ReadingListsOverflowView.kt

```diff
@@ -34,10 +35,12 @@ class ReadingListsOverflowView(context: Context) : FrameLayout(context) {
     init {
         binding.readingListsOverflowSelect.isVisible = ReleaseUtil.isPreBetaRelease
         binding.readingListsOverflowSortBy.setOnClickListener {
+            BreadCrumbLogEvent.logMenuItemSelection(context, "sort_lists_click")
```

Comment:
Similarly, in these events, there's no need to hard-code any names, since we know which View was clicked (`it`).
In fact, is there any reason we can't just say `BreadCrumbLogEvent.logClick(context, it)`?